modules/administration-guide/pages/creating-a-telemetry-plugin.adoc

```diff
@@ -405,7 +397,7 @@ spec:
       plugins:                           <2>
       - 'http://apache-che.apps-crc.testing/plugin.yaml'
 ----
-<1> The editorId to set default plug-ins for.
+<1> The editor Id to set default plug-ins for.
```

Comment:
```suggestion
<1> The link:https://www.eclipse.org/che/docs/stable/end-user-guide/selecting-a-workspace-ide/[editor `id`] to set the default plug-ins for.
```

Reply:
According to https://www.merriam-webster.com/dictionary/id:

id is an abbreviation for identification.

So either:

```suggestion
<1> The editor identification to set the default plug-ins for.
```

or:

```suggestion
<1> The editor identification to set the default plug-ins for.
```

but it's not code.